handbook/engineering/code-intelligence/goals.md

```diff
@@ -70,18 +70,18 @@ Progress on adoption and usage is tracked in our [Looker dashboard](https://sour
 **Milestones:**
 
 1. 🔄 Sourcegraph Cloud has up-to-date precise code intelligence for 5 Gradle repositories.
-    - 🔄 Manual configuration in a forked repository.
-    - 🔄 Gradle plugin implementation.
+    - ✅ Automatic inference for projects that use the Java plugin
+    - Automatic inference for projects that use the Android plugin
     - Publish Gradle plugin.
```

Comment:
The gradle/maven plugin may not be necessary, maybe we should remove them from the roadmap. I don't think we will implement plugins unless we see that they're necessary for customer adoption